<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SonnetUi</class>
 <widget class="QWidget" name="SonnetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="textLabel2">
     <property name="whatsThis">
      <string>&lt;qt&gt;&lt;p&gt;This word was considered to be an &quot;unknown word&quot; because it does not match any entry in the dictionary currently in use. It may also be a word in a foreign language.&lt;/p&gt;
&lt;p&gt;If the word is not misspelled, you may add it to the dictionary by clicking &lt;b&gt;Add to Dictionary&lt;/b&gt;. If you do not want to add the unknown word to the dictionary, but you want to leave it unchanged, click &lt;b&gt;Ignore&lt;/b&gt; or &lt;b&gt;Ignore All&lt;/b&gt;.&lt;/p&gt;
&lt;p&gt;However, if the word is misspelled, you can try to find the correct replacement in the list below. If you cannot find a replacement there, you may type it in the text box below, and click &lt;b&gt;Replace&lt;/b&gt; or &lt;b&gt;Replace All&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>Unknown word:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="m_unknownWord">
     <property name="toolTip">
      <string>Unknown word</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;&lt;p&gt;This word was considered to be an &quot;unknown word&quot; because it does not match any entry in the dictionary currently in use. It may also be a word in a foreign language.&lt;/p&gt;
&lt;p&gt;If the word is not misspelled, you may add it to the dictionary by clicking &lt;b&gt;Add to Dictionary&lt;/b&gt;. If you do not want to add the unknown word to the dictionary, but you want to leave it unchanged, click &lt;b&gt;Ignore&lt;/b&gt; or &lt;b&gt;Ignore All&lt;/b&gt;.&lt;/p&gt;
&lt;p&gt;However, if the word is misspelled, you can try to find the correct replacement in the list below. If you cannot find a replacement there, you may type it in the text box below, and click &lt;b&gt;Replace&lt;/b&gt; or &lt;b&gt;Replace All&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>&lt;b&gt;misspelled&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;Select the language of the document you are proofing here.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>&amp;Language:</string>
     </property>
     <property name="buddy">
      <cstring>m_language</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="m_contextLabel">
     <property name="toolTip">
      <string>Text excerpt showing the unknown word in its context.</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;Here you can see a text excerpt showing the unknown word in its context. If this information is not sufficient to choose the best replacement for the unknown word, you can click on the document you are proofing, read a larger part of the text and then return here to continue proofing.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>... the &lt;b&gt;misspelled&lt;/b&gt; word shown in context ...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QPushButton" name="m_addBtn">
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;The unknown word was detected and considered unknown because it is not included in the dictionary.&lt;br&gt;
Click here if you consider the unknown word not to be misspelled, and you want to avoid wrongly detecting it again in the future. If you want to let it remain as is, but not add it to the dictionary, then click &lt;b&gt;Ignore&lt;/b&gt; or &lt;b&gt;Ignore All&lt;/b&gt; instead.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>&lt;&lt; Add to Dictionary</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QListView" name="m_suggestions">
     <property name="toolTip">
      <string>Suggestion List</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;If the unknown word is misspelled, you should check if the correction for it is available and if it is, click on it. If none of the words in this list is a good replacement you may type the correct word in the edit box above.&lt;/p&gt;
&lt;p&gt;To correct this word click &lt;b&gt;Replace&lt;/b&gt; if you want to correct only this occurrence or &lt;b&gt;Replace All&lt;/b&gt; if you want to correct all occurrences.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="textLabel4">
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;If the unknown word is misspelled, you should type the correction for your misspelled word here or select it from the list below.&lt;/p&gt;
&lt;p&gt;You can then click &lt;b&gt;Replace&lt;/b&gt; if you want to correct only this occurrence of the word or &lt;b&gt;Replace All&lt;/b&gt; if you want to correct all occurrences.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="text">
      <string>Replace &amp;with:</string>
     </property>
     <property name="buddy">
      <cstring>m_replacement</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="m_replacement">
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;If the unknown word is misspelled, you should type the correction for your misspelled word here or select it from the list below.&lt;/p&gt;
&lt;p&gt;You can then click &lt;b&gt;Replace&lt;/b&gt; if you want to correct only this occurrence of the word or &lt;b&gt;Replace All&lt;/b&gt; if you want to correct all occurrences.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="Sonnet::DictionaryComboBox" name="m_language">
     <property name="toolTip">
      <string>Language Selection</string>
     </property>
     <property name="whatsThis">
      <string>&lt;qt&gt;
&lt;p&gt;Select the language of the document you are proofing here.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="3">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_suggestBtn">
       <property name="text">
        <string>S&amp;uggest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_replaceBtn">
       <property name="whatsThis">
        <string>&lt;qt&gt;
&lt;p&gt;Click here to replace this occurrence of the unknown text with the text in the edit box above (to the left).&lt;/p&gt;
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_replaceAllBtn">
       <property name="whatsThis">
        <string>&lt;qt&gt;
&lt;p&gt;Click here to replace all occurrences of the unknown text with the text in the edit box above (to the left).&lt;/p&gt;
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>R&amp;eplace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_skipBtn">
       <property name="whatsThis">
        <string>&lt;qt&gt;
&lt;p&gt;Click here to let this occurrence of the unknown word remain as is.&lt;/p&gt;
&lt;p&gt;This action is useful when the word is a name, an acronym, a foreign word or any other unknown word that you want to use but not add to the dictionary.&lt;/p&gt;
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>&amp;Ignore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_skipAllBtn">
       <property name="whatsThis">
        <string>&lt;qt&gt;
&lt;p&gt;Click here to let all occurrences of the unknown word remain as they are.&lt;/p&gt;
&lt;p&gt;This action is useful when the word is a name, an acronym, a foreign word or any other unknown word that you want to use but not add to the dictionary.&lt;/p&gt;
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>I&amp;gnore All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_autoCorrect">
       <property name="whatsThis">
        <string>&lt;qt&gt;
&lt;p&gt;Click here to let all occurrences of the unknown word remain as they are.&lt;/p&gt;
&lt;p&gt;This action is useful when the word is a name, an acronym, a foreign word or any other unknown word that you want to use but not add to the dictionary.&lt;/p&gt;
&lt;/qt&gt;</string>
       </property>
       <property name="text">
        <string>Autocorrect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sonnet::DictionaryComboBox</class>
   <extends>QComboBox</extends>
   <header>sonnet/dictionarycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_addBtn</tabstop>
  <tabstop>m_replacement</tabstop>
  <tabstop>m_suggestBtn</tabstop>
  <tabstop>m_replaceBtn</tabstop>
  <tabstop>m_replaceAllBtn</tabstop>
  <tabstop>m_skipBtn</tabstop>
  <tabstop>m_skipAllBtn</tabstop>
  <tabstop>m_suggestions</tabstop>
  <tabstop>m_language</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
